src/containers/AppBody.jsx

```diff
@@ -17,7 +18,8 @@ const styles = {
 class AppBody extends Component {
   componentDidMount() {
     const { dispatch, snippetKey } = this.props;
-    axios.get("../snippet.json")
+    const [user, snippet] = snippetKey.split("/");
```

Comment:
Right, but it still is a key that refers to a specific resource, so we do need to make sure that it is URI friendly.